samples/ControlCatalog/Pages/OpenGlPage.xaml.cs

```diff
@@ -256,7 +256,7 @@ private void CheckError(GlInterface gl)
 
         protected unsafe override void OnOpenGlInit(GlInterface GL, int fb)
         {
-            CheckError(GL);
+            OpenGlPageControl.CheckError(GL);
```

Comment:
Not sure why we need static class name here and in some other places, as it's used in the same scope.

Reply:
@workgroupengineering have you checked why we need class name here? There quite a lot of changes in this PR already, but this specifically doesn't seem to be needed.

Reply:
I did, probably when I have merge I made some mistakes.